tkg/managementcomponents/helper.go

```diff
@@ -165,14 +166,15 @@ func setAkoOperatorConfig(tkgPackageConfig *TKGPackageConfig, userProviderConfig
 				AviManagementClusterControlPlaneVipNetworkName: convertToString(userProviderConfigValues[constants.ConfigVariableAviManagementClusterControlPlaneVipNetworkName]),
 				AviManagementClusterControlPlaneVipNetworkCIDR: convertToString(userProviderConfigValues[constants.ConfigVariableAviManagementClusterControlPlaneVipNetworkCIDR]),
 				AviControlPlaneHaProvider:                      convertToBool(userProviderConfigValues[constants.ConfigVariableVsphereHaProvider]),
+				AviIngressNodeNetworkList:                      convertToString(userProviderConfigValues[constants.ConfigVariableAviIngressNodeNetworkList]),
```

Comment:
If it is not empty. you need to convert it into JSON string, not just convert to String

Reply:
`[{"networkName":"", "cidr":["10.1.1.1/24"]}]`